Makefile

```diff
@@ -16,7 +16,6 @@ protos:
 	./tools/GENERATE-RPC.sh
```

Comment:
Can you not include this change in the PR?

This might be because of a stale local copy.